src/widgets/BugsHistory/components/BugsHistory.scss

```diff
@@ -5,4 +5,8 @@
     font-size: $font-big;
     color: $rocket-gray;
   }
+
+  .chart-container {
```

Comment:
Will it work still, if you add the `overflow: hidden;` to the `.panel` class? It would be better, imho, as it will keep all future widgets in check.